### ISSUE TITLE
When computer cars resets under a bridge, it gets confused

### ISSUE THREAD

**Issue by Wuzzy2** [juzzlin/DustRacing2D#51]
It is possible to drive through the little ramp of the bridge when your car is under the bridge.

How to reproduce:
* Create a test map to force computer cars under various bridges. The bridges must face North/South, while the road below the road must face West/East
* Have a waypoint exactly on the bridge tile
* Make it so the computer cars crash in an unfortunate fashion when driving under the bridge
* If the cars reset, you did it right

EXPECTED: The restting computer cars will respawn in the direction of the lower road and under the bridge.

ACTUAL: The resetting cars will respawn in the direction of the lower road, but ON the bridge. The poor fellows will perpetually drive into the bridge railing, then reset, drive into the railing again, and so on, in an infinite loop and just fail to continue the race at all.

**Comment by juzzlin** [comment]
Yeah, this can happen because there are no layer support in the waypoints :)
